pkg/corerp/datamodel/environment.go

```diff
@@ -123,3 +128,29 @@ type ProvidersAWS struct {
 	// Scope is the target level for deploying the aws resources
 	Scope string `json:"scope,omitempty"`
 }
+
+// EnvironmentVariables describes structure enabling environment variables to be set.
+type EnvironmentVariables struct {
+	// Additional properties for the resource
+	AdditionalProperties map[string]any `json:"additionalProperties,omitempty"`
```

Comment:
pls update to map[string]string

Reply:
So this one is different I guess https://github.com/radius-project/radius/blob/main/pkg/corerp/datamodel/extender.go#L73

Reply:
Discussed offline